dev/app-shell/elements/arc-app.html

```diff
@@ -566,6 +566,18 @@
     _onSuggest(e, suggestion) {
       this._setState({plan: suggestion.plan});
     }
+    _onMakeSuggestions(e) {
+      let search = e.detail.search;
+      if (search)
+        search = search.trim().toLowerCase();
+      if (!search)
```

Comment:
All if-statements in this code are followed by a block (`{ }`). This is just something I do for my own readability. No big deal, but I'll probably change it out of habit if I see it later, lol.

Reply:
done